README.md

```diff
@@ -95,3 +95,12 @@ Finally, "yank" out interesting nodes based on their flavors and scores. For exa
 ## More Examples
 
 Our docs are a little sparse so far, but [our tests](https://github.com/mozilla/fathom/tree/master/test) might help you in the meantime.
+
+## Running Fathom on the Client
+
+Out of the box, Fathom assumes your running the library in a Node.js project. If you're interested in running Fathom on the client side, you can run:
+
+`npm run clientize`
+
+This command will create a file named _fathom.bundle.js_ under the bin/ directory that you can import into your web page using a <script> tag.
```

Comment:
the `<script>` tag needs to be wrapped in backticks/code format, since it seems to be swallowing the rendering of the rest of the page.